packages/x-components/src/x-modules/url/components/url-handler.vue

```diff
@@ -165,6 +165,27 @@
       this.urlLoaded = true;
     }
 
+    /**
+     * Detects if the user typing or pasting special/forbidden characters in some URL param.
+     *
+     * @internal
+     * @param urlValue - Param from the url that will be checked for special characters.
+     * @returns Param without special characters.
+     */
+    protected hasSpecialKeys(urlValue: string): string {
```

Comment:
2 days ago this function was testing if the URL had special keys to not emit the event.

Now the function has changed to replace the characters...so I would change to this:

```suggestion
    /**
     * Detects if the user typing or pasting special/forbidden characters in some URL param and removes them
     *
     * @internal
     * @param urlValue - Param from the url that will be checked for special characters.
     * @returns Param without special characters.
     */
    protected removeSpecialKeys(urlValue: string): string {
```